userscripts/user_exp_time_matrix.js

```diff
@@ -0,0 +1,453 @@
+/**
+ * Explorer Time Matrix for TSO client
+ * v1.0
+ *
+ * Created by
+ * .▄▄ ·  ▄▄▄· .▄▄ ·     .▄▄ ·  ▄▄▄· • ▌ ▄ ·.
+ * ▐█ ▀. ▐█ ▀█ ▐█ ▀.     ▐█ ▀. ▐█ ▀█ ·██ ▐███▪
+ * ▄▀▀▀█▄▄█▀▀█ ▄▀▀▀█▄    ▄▀▀▀█▄▄█▀▀█ ▐█ ▌▐▌▐█·
+ * ▐█▄▪▐█▐█ ▪▐▌▐█▄▪▐█    ▐█▄▪▐█▐█ ▪▐▌██ ██▌▐█▌
+ *  ▀▀▀▀  ▀  ▀  ▀▀▀▀      ▀▀▀▀  ▀  ▀ ▀▀  █▪▀▀▀
+ *
+ * You can find me on Discord (sas_sam).
+ *
+ * If you like my work, you can support me: https://ko-fi.com/sas_sam
+ *
+ */
+var ExplorerTimeMatrix_PFB_RB_STATUS = false;
+var ExplorerTimeMatrix_ADV_RB_STATUS = false;
+
+function logger(message) {
```

Comment:
client already have function "debug" with same logic.. use it

Reply:
removed